sdcm/tester.py

```diff
@@ -615,25 +622,32 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
 
         self.init_resources()
 
-        if self.db_cluster and self.db_cluster.nodes:
```

Comment:
This really complicates the `setUp` function, if someone not familiar with the multitenant setup would need to add something here, he'll probably break multitenant handling.

Reply:
This change keeps the logic for the single cluster setup adding possibility to run multiple clusters one.
Then SCT gets broken everywhere constantly, all the time, we do not need to have multi-tenant setup fo rit.
So, I don't see any reason to look for other solution instead of this working one.

Reply:
@enaydanov calling you for second opinion on this one, or maybe you have other ideas...